### ISSUE TITLE
[Feature Request] Allow disable link opening warning

### ISSUE THREAD

**Issue by soredake** [telegramdesktop/tdesktop#25397]
### Is your feature request related to a problem?

Right now, if you try to open formatted link, you will this annoying un-skippable warning:
![Снимок экрана 2022-11-20 163146](https://user-images.githubusercontent.com/5204968/202908087-72c34543-dd9a-4fe4-ace0-106fa979056e.png)

### Describe the solution you'd like

Have a checkbox that will disable this warning, for example, you can disable warning while opening `.exe` (which i think is more dangerous that just opening a link) files in telegram desktop:
![Снимок экрана 2022-11-20 162848](https://user-images.githubusercontent.com/5204968/202908211-642f2a54-5e04-4156-abdf-5ab662c2ea30.png)

### Describe alternatives you've considered

Living with this annoying warning.

### Additional context

With such option added, i can decide myself if this warning is worth it or not.

Example implementation from 64gram https://github.com/TDesktop-x64/tdesktop/commit/00f10f305bf8201fd821bb943cdc75d238fbaebe

**Comment by Aokromes** [comment]
imho not desired way, instead do you trust this domain?
also for exe files the same do you trust this source? (by channel id / group id / user id)

**Comment by soredake** [comment]
>imho not desired way, instead do you trust this domain?

Maybe, but not for me, i don't want to have this warning at all, telegram don't need to protect me from myself, i'm fine if there will be very big warning for disabling it completely, if fine even with hidden setting somewhere.

**Comment by simonbcn** [comment]
I totally agree, it's a pain the confirmation windows with every link clicked.
